docs/python_docs/python/tutorials/getting-started/crash-course/2-create-nn.md

```diff
@@ -21,7 +21,7 @@ npx.set_np()  # Change MXNet to the numpy-like mode.
 ## Create your neural network's first layer
```

Comment:
This notebook should at least highlight the usage of Parameter class and how to get the arrays from that class inside a Block. cc @leezu